oxidation/libparsec/crates/client_connection/src/error.rs

```diff
@@ -67,6 +67,43 @@ pub enum CommandError {
     WrongApiVersion(String),
 }
 
+impl PartialEq for CommandError {
```

Comment:
Add `// Custom equality to skip comparison of some fields`

Reply:
Done